src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
```

Comment:
The image element `.card__img` should have `display: block;` to remove the default inline behavior that can cause extra space below the image as per [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).